attestation-agent/attester/src/csv/mod.rs

```diff
@@ -48,14 +43,17 @@ impl Attester for CsvAttester {
         report_data.resize(64, 0);
 
         let data = report_data.as_slice().try_into()?;
-        let mut csv_guest = CsvGuest::open().unwrap();
+        let mut csv_guest = CsvGuest::open()?;
 
-        let (attestation_report, report_signer) = csv_guest.get_report(Some(data), None).unwrap();
+        let (attestation_report, report_signer) = csv_guest.get_report(Some(data), None)?;
 
-        let cert_data = download_hskcek_from_kds(&report_signer.sn).await?;
+        // Fill in HSK_CEK with 0
+        let cert_size =
+            std::mem::size_of::<ca::Certificate>() + std::mem::size_of::<csv::Certificate>();
+        let cert_data = vec![0; cert_size];
         let mut cert_data = &cert_data[..];
-        let hsk = ca::Certificate::decode(&mut cert_data, ()).unwrap();
-        let cek = csv::Certificate::decode(&mut cert_data, ()).unwrap();
+        let hsk = ca::Certificate::decode(&mut cert_data, ())?;
+        let cek = csv::Certificate::decode(&mut cert_data, ())?;
```

Comment:
Maybe it would be better to make these members of the CertChain struct optional rather than create these empty certs.

Also, it might make sense to allow the attester to download the certs optionally. For some users I actually think the VM will be more access to the inernet than Trustee will.